android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/screens/MeasureReportResultScreen.kt

```diff
@@ -81,6 +82,7 @@ fun MeasureReportResultScreen(
   )
 }
 
+@SuppressLint("UnusedMaterialScaffoldPaddingParameter")
```

Comment:
Apply the padding as suggested, remove this suppression. See how this is used in [RegisterScreen#L111](https://github.com/opensrp/fhircore/blob/7781e177fc4a60216172d1893fa64c883b1f8b82/android/quest/src/main/java/org/smartregister/fhircore/quest/ui/register/RegisterScreen.kt#L111)